Analytics/Classes/SEGAnalyticsConfiguration.h

```diff
@@ -89,21 +89,21 @@ typedef NSMutableURLRequest * (^SEGRequestFactory)(NSURL *);
 /**
  * Dictionary indicating the options the app was launched with.
  */
-@property (nonatomic, assign) NSDictionary *launchOptions;
+@property (nonatomic, strong, nullable) NSDictionary *launchOptions;
```

Comment:
@f2prateek calling particular attention to this one. it used to be `assign` which isn't right.